src/bot.py

```diff
@@ -1,71 +1,76 @@
-import math
-
 from rlbot.agents.base_agent import BaseAgent, SimpleControllerState
+from rlbot.messages.flat.QuickChatSelection import QuickChatSelection
 from rlbot.utils.structures.game_data_struct import GameTickPacket
 
-from util.orientation import Orientation
+from util.ball_prediction_analysis import find_slice_at_time
+from util.drive import steer_toward_target
+from util.sequence import Sequence, ControlStep
 from util.vec import Vec3
 
 
 class MyBot(BaseAgent):
 
-    def initialize_agent(self):
-        # This runs once before the bot starts up
-        self.controller_state = SimpleControllerState()
```

Comment:
I don't think we should delete this one.

Reply:
I got rid of self.controller_state as an instance variable because it's a common source of bugs, e.g. people get their handbrake stuck on True for many frames without realizing it.

I did not think it was important to keep the existence of initialize_agent highly visible because recent changes to hot reload have made it less necessary. For example, I suspect that pytorch can now be imported normally with no issues.

Reply:
Okay, great. Then it is not a problem. But this is still the place to call `get_field_info()` right?

Reply:
True, that's a sensible place to do it. Maybe I should add some basic boost parsing functionality to either the example bot or the framework